### PR TITLE
checkmetrics: Add checkmetrics makefile and documentation

### LINKED ISSUE #7172: checkmetrics: add makefile for kata metrics tests

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds checkmetrics documentation and makefile which is used to process the metrics json results files.

Fixes #7172

### CODE REVIEW

**Review by dborquez — APPROVED** [review]

File: tests/metrics/README.md [hunk 1]
```diff
@@ -190,3 +190,7 @@ happen if `JSON_URL` is set. The settings are not mutually exclusive, and both c
 set if necessary.
 
 `JSON_TX_ONELINE` applies to both types of transmission.
+
+## `checkmetrics`
+
+For further reference see the [`checkmetrics`](cmd/checkmetrics).
```
Comment (chavafg) [inline tests/metrics/README.md:196]:
```suggestion
`checkmetrics` is a CLI tool to check a metrics CI results file. For further reference see the [`checkmetrics`](cmd/checkmetrics).
```

Description taken from https://github.com/kata-containers/tests/blob/main/cmd/README.md

Comment (GabyCT) [reply tests/metrics/README.md]: @chavafg changes applied

**Review by chavafg — APPROVED** [review]

**Pull request merged**